### PR TITLE
astgen: Enforce the use of aliased operand accessors

### PR DESCRIPTION
**Pull request by gezalore**
This patch enforces the use of the most specific accessors for operands which have an '@astgen alias' declaration, by making the superclass accessors of the same operands private. This ensures client code is cleaner as you can't use multiple different methods to reference the same operands (which we used to in some places). Also prep for some refactoring.

### CODE REVIEW

File: src/V3Const.cpp
Comment (gezalore) [inline src/V3Const.cpp:3699]: Was/is this a bug here? lhsp and fromp are the same node, cant be both types.

Comment (wsnyder) [reply src/V3Const.cpp:3699]: These are looking for a select where the bit to select from is a constant.  Please make a pre-pull to fix this as it's an algorithm change, feel free to merge if it passes.  Likewise please prepull (together or separate) any other algorithm changes because of this good cleanup.

**Review by wsnyder — APPROVED** [review]
Feel free to squash-merge this once the algorithm comment is fixed and merged.  (Or merge this with that broken check commented out and fix it as a later pull).

File: src/V3Const.cpp
Comment (gezalore) [reply src/V3Const.cpp:3699]: Ok, what I have here still has the bug so this is not a fix for the algorithm. I will do a new pull after to fix that.

**Pull request merged**
